src/main/java/at/hannibal2/skyhanni/tweaker/SkyHanniTweaker.java

```diff
@@ -0,0 +1,14 @@
+package at.hannibal2.skyhanni.tweaker;
+
+import org.spongepowered.asm.launch.MixinTweaker;
+
+@SuppressWarnings("unused")
+public class SkyHanniTweaker extends MixinTweaker {
```

Comment:
Please use `(List<String>) Launch.blackboard.get("TweakClasses")` to add the mixin tweaker class name (as a string) to the list of tweakers to process, instead of extending MixinTweaker. While the MixinTweaker itself has checks to prevent double initializations to some degree, other tweakers do not, and this sets a bad example.